### ISSUE TITLE
How to turn off the concent hello message on the console?

### ISSUE THREAD

**Issue by ddatall** [concentjs/concent#46]
`------------ CC TIP ------------`
<kbd>cc version 2.14.14</kbd>

[util.js#L211](https://github.com/concentjs/concent/blob/master/src/support/util.js#L211)

Can't find options to turn off.

**Comment by fantasticsoul** [comment]
```
import { run } from 'concent';

run(models, {
  log: false, // no log behavior
});
```

**Comment by ddatall** [comment]
no, I need to keep error logging.

**Comment by ddatall** [comment]
> ```
> import { run } from 'concent';
> 
> run(models, {
>   log: false, // no log behavior
> });
> ```

no, I need to keep error logging.

**Comment by fantasticsoul** [comment]
I have just released a new version `2.14.18`, allow user set param `logVersion` to turn of loggin version
```js
import { run } from 'concent';

run(models, {
  logVersion: false, // default ist true
});
```
